packages/app-store/plausible/zod.ts

```diff
@@ -4,9 +4,7 @@ import { eventTypeAppCardZod } from "../eventTypeAppCardZod";
 
 export const appDataSchema = eventTypeAppCardZod.merge(
   z.object({
-    // FIXME: Due to some reason autogenerated zod schema at prisma/zod/eventtype.ts fails typecheck on using `.default()`. It somehow thinks that trackingId can be undefined
```

Comment:
The reason actually was that when using default you have to mark it optional as well which is weird.